graph/typeDefs.graphql

```diff
@@ -631,6 +631,20 @@ type SetCommentStatusResponse implements Response {
   errors: [UserError]
 }
 
+# Response to addCommentTag mutation
+type AddCommentTagResponse implements Response {
+  # An array of errors relating to the mutation that occured.
+  comment: Comment
+  errors: [UserError]  
+}
+
+# Response to removeCommentTag mutation
+type RemoveCommentTagResponse implements Response {
+  # An array of errors relating to the mutation that occured.
+  comment: Comment
```

Comment:
As we discussed on Slack, I've left this in this api definition, but lifted the findByIds to power it as high as possible.

Once the permalink PR lands with its [comment query](https://github.com/coralproject/talk/pull/352/files#diff-12b7fbc29e00b65c215fd45aea1ba81bR484), it'll be easier to just have the client re-fetch the comment via that query, and remove the utility of returning it here. But let's make that improvement to a later PR.